tests/common/snappi_tests/snappi_helpers.py

```diff
@@ -182,17 +182,29 @@ def get_ports(self, peer_device=None):
         retval = []
         for port in self.current_snappi_port_list:
             info_list = port.split('/')
-            dict_element = {
-                'ip': info_list[0],
-                'card_id': info_list[1].replace('Card', ''),
-                'port_id': info_list[2].replace('Port', ''),
-                'peer_port': info_list[3],
-                'peer_device': info_list[4],
-                'speed': info_list[5]
-            }
+            if 'Card' in port:
```

Comment:
@sdszhang: please not this change . After this PR we need make sure in links.csv we have to define line this `Card1/Port9`